packages/sdk/client/src/client/client.ts

```diff
@@ -213,6 +213,14 @@ export class Client {
     return this._runtime.mesh;
   }
 
+  /**
+   * EDGE client.
+   */
+  get edge(): EdgeHttpClient {
```

Comment:
This is a leaky abstraction, I believe Dima added _edgeClient to Client as a quick hack. It will be moved into  ClientServices in the near future, so it is better to not expose it like that. cc @dmaretskyi

Reply:
my impression was actually the opposite of that: that edge client was already in client services but needed to be exposed on the client which is why it was added here.

The reason for this is that currently we have this `useEdgeClient` hook which is creating new edge clients all over the place, having it in client at least centralizes this so there's only two (one on frontend, one in services). If we wanted to take it a step further we could then replace the edge on the frontend client with a proxy, but maintain the interface, however I don't think that should stop us from moving usage from separate edge clients to this one in the short term.

Reply:
Yes, it is better than what we had before. Could you add an experimental/deprecated notice in JSDoc?